spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -129,8 +129,8 @@ def resolution_center
         client.get_resolution_center(apple_id, platform)
       end
 
-      def ratings
-        attrs = client.get_rating_summary(apple_id, platform)
+      def ratings(version_id: '', storefront: '')
+        attrs = client.get_rating_summary(apple_id, platform, version_id, storefront)
         attrs[:application] = self
         Tunes::AppRatings.factory(attrs)
```

Comment:
Also, ideally I think this would be a class level call on `Tunes::AppRatings` that way the `Application` class is not responsible for parsing data about the ratings, but that is something we can move in the future.